mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/StoredRecordChunksKafkaHandler.java

```diff
@@ -102,8 +117,17 @@ public Future<String> handle(KafkaConsumerRecord<String, String> record) {
 
           LOGGER.debug("handle:: RecordsBatchResponse has been received, starting processing chunkId: {} chunkNumber: {} jobExecutionId: {}", chunkId, chunkNumber, jobExecutionId);
           saveCreatedRecordsInfoToDataImportLog(storedRecords, okapiConnectionParams.getTenantId());
-          return recordsPublishingService.sendEventsWithRecords(storedRecords, jobExecutionId,
-              okapiConnectionParams, eventType.value())
+          return jobExecutionService.getJobExecutionById(jobExecutionId, okapiConnectionParams.getTenantId())
+            .compose(jobExecutionOptional -> {
+              if (jobExecutionOptional.isPresent()) {
```

Comment:
@VRohach , could you add some debug information here and in other places with new changes?

Reply:
@Aliaksandr-Fedasiuk  Done.